docs/developer-docs/integrations/bitcoin/bitcoin-how-it-works.md

```diff
@@ -2,105 +2,93 @@
 
 ## Overview
 
-Two advanced engineering challenges needed to be solve to enable Bitcoin smart contracts on ICP: 
-- A protocol-level integration of ICP with the Bitcoin network.
-- A chain-key signatures based on a novel threshold ECDSA protocol.
 
-This page provides a general overview of the Bitcoin integration technology. For a deeper technical explanation of ckBTC, please see the [wiki page](https://wiki.internetcomputer.org/wiki/Chain-key_Bitcoin).
+The Internet Computer Protocol has been integrated with the Bitcoin network to obtain a direct technical integration between the two. 
```

Comment:
_"has been integrated ... to obtain ... integration"_ sounds a bit redundant and doesn't give much information. I would go with something more direct and less formal:
"Smart contracts running on IC can directly interact with the Bitcoin network without any bridges".